python/servo/testing_commands.py

```diff
@@ -747,3 +749,51 @@ def smoketest(self, build_type: BuildType, params):
         # of panics won't cause timeouts on CI.
         return self.context.commands.dispatch('run', self.context, build_type=build_type,
                                               params=params + ['-f', 'tests/html/close-on-load.html'])
+
+    @Command('try', description='Runs try jobs by force pushing to try branch', category='devenv')
+    @CommandArgument('--remote', '-r', default="origin", help='A git remote to run the try job on')
+    @CommandArgument('try_string', default=None, nargs='...',
+                     help="A list of try strings specifying what kind of job to run.")
+    def try_command(self, remote="origin", try_string=None):
```

Comment:
```suggestion
    def try_command(self, remote="origin", try_string=["full"]):
```

Reply:
I've moved this and the other default to `@CommandArgument`.